versions/versions.gradle

```diff
@@ -7,27 +7,27 @@
 ext.CONFIG.versions = [
     // Java compatibility target version
     java      : '1.7',
-    kotlin    : '1.2.0',
+    kotlin    : '1.3.21',
 
     // Android plugins, tools, SDK and libraries
     android   : [
         libraries: [
-            playServices: '11.0.4',
-            support     : '26.0.0',
+            playServices: '16.0.0',
+            support     : '28.0.0',
```

Comment:
Similar to play services updating the support library to 28.0.0 creates incompatibilities. For the SDKs we will probably stay on 26 until we switch to androidx - or what do you think @corycaywood ?

Reply:
Yeah, good point. We should probably leave it at 26 and start switching to androidx libraries.